projects/Canaille/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+
+{
+  metadata = {
+    summary = "Canaille is a zero-knowledge opinionated identity server.";
+    subgrants = [
+      "Canaille"
+    ];
+  };
+
+  nixos.modules.services = {
+    Canaille = {
```

One last change, but this looks good otherwise.

```suggestion
    canaille = {
```

---

I have resolved, thank you!